File: testresources/csharp/benchmark/Domain/Direct/Violating/CallConstructor_GenericType_MultipleTypeParameters.cs
using System.Collections.Generic.Dictionary;

namespace Domain.Direct.Violating
{
	public class CallConstructor_GenericType_MultipleTypeParameters
	{
		public CallConstructor_GenericType_MultipleTypeParameters()
		{
            dictionary = Dictionary<ProfileDAO, UserDAO>();
		}
	}
}

File: src/test/resources/java/accuracy/src/domain/direct/allowed/CallConstructor_GenericType_MultipleTypeParameters.java
package domain.direct.allowed;

import java.util.HashMap;

import domain.direct.Base;
import technology.direct.dao.ProfileDAO;
import technology.direct.dao.UserDAO;

public class CallConstructor_GenericType_MultipleTypeParameters extends Base {

	public void CallConstructor(){
		hashMap = new HashMap<ProfileDAO, UserDAO>();
	}
}